### PR TITLE
Capture data in a database

### PR DESCRIPTION
**Pull request by robbiepaul**
- This PR adds sqlite as a database adapter and ActiveRecord as our ORM
- Added a basic migration to create a `payment` table so we can track payments locally
- Creates a payment record when the user clicks 'Pay now'
- Updates the payment record when they hit the `return_url`
- Only shows 'payment successful' message if GOV Pay sends back a status of `success`

### CODE REVIEW

**Comment by robbiepaul** [comment]
[Capturing data in a database](https://trello.com/c/DlbVJaa3/5-capturing-data-in-a-database)

File: views/payment.haml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- if @payment.successful?
+  .govuk-panel.govuk-panel--confirmation
+    %h2.govuk-panel__title
+      Payment complete
+    .govuk-panel__body
+      Your reference number
+      %br
+      %strong= @payment.govpay_reference
+  %p
```
Comment (deberny) [inline views/payment.haml:9]: should the %p be moved out of if / else, so that the 'back link' appears regardless ? A tiny thing :)

Comment (robbiepaul) [reply views/payment.haml:9]: I think we should create a separate card for looking at the whole flow. Logically I think it makes sense to put a 'Try again' call to action here instead but we should chat about it in stand up

Comment (deberny) [reply views/payment.haml:9]: ok, that's cool

**Pull request merged**